examples/build/main.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	v1 "github.com/openshift/api/build/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	buildv1 "github.com/openshift/client-go/build/clientset/versioned/typed/build/v1"
+	"path/filepath"
+	"flag"
+	"k8s.io/client-go/tools/clientcmd"
+	"os"
+	"fmt"
+)
+
+func main() {
```

Comment:
Have your main call a function that returns an error.  It makes the code below copy/pasteable if you do that.

Reply:
Done